### DISCUSSION TITLE
Aliases for enums?

### DISCUSSION THREAD

**Discussion by sotnii** [aarondl/sqlboiler#1260] [Q&A]
I have psql enum called `oauth_provider`. Sqlboiler generates type called `OauthProvider`. I want it to be called `OAuthProvider`. How can I achieve this?

**Comment by stephenafamo** [comment]
Right now there are no aliases for Enums.

I am not 100% sure but it may be possible to get SQLBoiler to interpret `Oauth` as `OAuth` using the [inflections configuration](https://github.com/volatiletech/sqlboiler#inflections)
